frontend/src/pages/operation_show/evidence_modals/index.tsx

```diff
@@ -90,6 +90,18 @@ export const CreateEvidenceModal = (props: {
   const [selectedCBValue, setSelectedCBValue] = React.useState<string>(evidenceTypeOptions[0].value)
   const getSelectedOption = () => evidenceTypeOptions.filter(opt => opt.value === selectedCBValue)[0]
 
+  // Validation logic for description and content fields
+  const validateForm = () => {
+    const errors = []
+    if (descriptionField.value.trim() === "") {
+      errors.push("Description cannot be empty.")
+    }
+    if (getSelectedOption().value !== 'event' && binaryBlobField.value === null && codeblockField.value.code.trim() === "") {
+      errors.push("Evidence content cannot be empty.")
+    }
+    return errors
+  }
```

Comment:
I need to run this, but docker seems to be misconfigured on this box. But, I think this will be okay.  I don't think we have a lot of validation rules elsewhere to pull from, but I'll take a look to see if we solved this in a different way elsewhere.